### PR TITLE
Update code for beta3

### PR DESCRIPTION
**Pull request by tractorcow**
Story: https://github.com/silverstripeltd/open-sourcerers/issues/19

### CODE REVIEW

File: mysite/javascript/calltoaction.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-            '<h3>Like what you see?</h3>' +
-            '</div>');
-
-
-    })
+(function ($) {
+  $(document).ready(function () {
+    $('#cms-menu').find('.cms-panel-content').append('' +
+      '<div class="demo-ctas">' +
+      '<a onClick="ga(\'send\', \'event\', \'demo-cms\', \'cta-click\', \'learn-cms\'); ga(\'send\', \'pageview\', \'/admin/cta-click/\');" href="https://userhelp.silverstripe.org/en/3.3/?utm_source=demo-cms&utm_medium=cta-click&utm_content=learn-cms&utm_campaign=demo-improvements" target="_blank" class="cta-btn">Learn the CMS</a>' +
```
Comment (robbieaverill) [inline mysite/javascript/calltoaction.js]: Should we maybe remove the version number here and leave it to open the latest stable version?

Comment (tractorcow) [reply mysite/javascript/calltoaction.js]: I just reformatted the file I'm not upgrading the code itself.

Comment (robbieaverill) [reply mysite/javascript/calltoaction.js]: Yeah, ok

File: mysite/code/DemoResetTask.php [hunk 2]
```diff
@@ -1,47 +1,71 @@
...
+            }
+        }
+
+        // Use `which`
+        $path = exec("which $bin 2>/dev/null");
+        if ($path) {
+            return $path;
+        }
+
+        throw new Exception("Could not find bin path for command $bin");
```
Comment (robbieaverill) [inline mysite/code/DemoResetTask.php]: You'll probably need to import this

**Comment by tractorcow** [comment]
Deployed to http://demov4-uat.sites.silverstripe.com/admin/assets/ and seems to work fine

File: mysite/code/DemoResetTask.php (hunk 2 above)
Comment (tractorcow) [reply mysite/code/DemoResetTask.php]: This class isn't in a namespace so it's unnecessary.

File: mysite/javascript/calltoaction.js (hunk 1 above)
Comment (tractorcow) [reply mysite/javascript/calltoaction.js]: I've removed the version anyway. :)

**Comment by tractorcow** [comment]
ta!

**Pull request merged**